src/thumbnail/thumbnail-info.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+class ThumbnailInfo {
+  _url: string;
+  _width: number;
+  _height: number;
+  _x: number;
+  _y: number;
+
+  set url(value: string) {
```

Comment:
better to set all in the constructor and expose gets only

Reply:
Prefer to keep it like that

Reply:
can you explain? as I see in dash and kaltura-player, they can pass these params to the ctor 
why not keep this api read only?